### PR TITLE
Insurance Information Form: Utilize the `itemAriaLabel` function in the form's schema to append extra text to the "edit" button's aria-label attribute

### PR DESCRIPTION
**Pull request by obliviga**
## Description
Utilize the `itemAriaLabel` function in the form's schema to append extra text to the "edit" button's aria-label attribute

## Original issue(s)
https://github.com/department-of-veterans-affairs/va-forms-system-core/issues/358

## Testing done
See screenshots below

## Screenshots
- Only policy number field filled out: image
- Only group code field filled out: image
- Both policy number and group code fields filled out (will only show policy number in the aria-label attribute): image

## Acceptance criteria
- [x] An engineer is able to utilize the `itemAriaLabel` function to append to the Edit button's aria-label attribute in any form that is generated by the `ArrayField`.

## Definition of done
- [x] Feature is covered by a unit test
- [x] Documentation has been updated: https://vfs.atlassian.net/wiki/spaces/pilot/pages/2104623233/VA+Forms+Library+-+About+schema+and+uiSchema#ui%3Aoptions
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/forms-system/src/js/fields/ArrayField.jsx
Comment (micahchiang) [inline src/platform/forms-system/src/js/fields/ArrayField.jsx:353]: I love the nullish coalescing operator 😎 

I do think we want to keep the itemName only option, since we still need an aria label in the event a uiSchema doesn't contain a supplementalLabel

Comment (micahchiang) [reply src/platform/forms-system/src/js/fields/ArrayField.jsx:353]: Oh wait. I misread this lol. This works just fine 🎉

**Review by bjmccotter7192 — COMMENTED** [review]
LGTM

**Review by bkjohnson — APPROVED** [review]

**Review by micahchiang — APPROVED** [review]

**Comment by micahchiang** [comment]
@lzapson - are you part of either bam1 or caregiver codeowner teams? I think this PR needs an approval from one of those teams.

**Review by lzapson — APPROVED** [review]
The changes to general.js, which are in the files of the hca application my team maintains, LGTM

File: src/platform/forms-system/src/js/fields/ArrayField.jsx
Comment (jerekshoe) [inline src/platform/forms-system/src/js/fields/ArrayField.jsx:13]: This should probably be:
```js
import set from 'platform/utilities/data/set';
```

**Review by jerekshoe — APPROVED** [review]

**Pull request merged**
